_posts/2018/01/2018-01-17-sponsor-carpentrycon.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+authors: ["Belinda Weaver"]
+title: "An Open Letter to Companies - Sponsor CarpentryCon 2018"
```

Comment:
This post is great and relevant to any organization that would want to support CarpentryCon. Could the title be: "Why Become a CarpentryCon Sponsor?" like for PyCon https://us.pycon.org/2017/sponsors/why-sponsor/